pkgs/development/interpreters/erlang/27-rc2.nix

```diff
@@ -0,0 +1,7 @@
+{ mkDerivation }:
+
+mkDerivation {
+  version = "27.0-rc2";
+  sha256 = "sha256-o+uqQMlrh7wJQbiecMnn3/tv6nqSJh7unOj0A95mMb0=";
```

Comment:
not sure where `mkDerivation` comes from, but prefer hash over sha256 for SRI hash notation

Reply:
I copied this style from https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/interpreters/erlang/26.nix

Do you have a suggestion for what else this could be? Just doing s/sha256 =/hash =/ doesn't work.

Reply:
Found it, https://github.com/NixOS/nixpkgs/blob/ee64a5f3ea5ea4c4ef7820a9bda41c16cd1c9ae4/pkgs/development/beam-modules/lib.nix#L29-L38

I guess this is passing the builder into 27-rc2.nix and then overriding parameters. Those parameters include `sha256` and `version`!

Reply:
We could change https://github.com/NixOS/nixpkgs/blob/ee64a5f3ea5ea4c4ef7820a9bda41c16cd1c9ae4/pkgs/development/interpreters/erlang/generic-builder.nix#L36

To use `hash` instead of `sha256` and then update all of the 27-rc2.nix-like files.

I probably shouldn't do that in this PR though?

Reply:
thanks for the investigation, no need to do part of this PR